src/widget/wsearchlineedit.cpp

```diff
@@ -1,14 +1,13 @@
 #include "wsearchlineedit.h"
 
-#include <qcompleter.h>
-#include <qstringliteral.h>
-
 #include <QAbstractItemView>
 #include <QApplication>
+#include <QCompleter>
```

Comment:
Oh, during my previous review I didn't notice this is already included in wsearchlineedit.h
Please remove it here, git commit --amend (maybe name it more explicitely) and push one last time.

FYI rebasing after a review _without approval_ is not appreciated since that means someone has to review again, instead of just checking the last commits. Here, it's not much of a drama because the change set is rather small and can be verified by manual testing.

Reply:
> FYI rebasing after a review without approval is not appreciated

Ah sorry, I'll try to keep that in mind for the next time

Reply:
I didn't neither review again nor test manually, so if you like to --fixup previous commits with the latest changes and squash..  feel free.

Reply:
I went with amending for now, most of these are just small stylistic changes anyway